src/index.html

```diff
@@ -0,0 +1,110 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="./output.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap"
+    rel="stylesheet">
+</head>
+
+<body class="flex flex-col h-screen justify-between">
+  <!-- header -->
+  <div class="hidden lg:flex justify-end items-center gap-3 p-2 text-sm w-full">
+    <div class="flex gap-2">
+      <a href="" class="hover:text-black hover:underline">Gmail</a>
+      <a class="hover:text-black hover:underline">images</a>
+    </div>
+    <div class="p-2 hover:rounded-full hover:bg-gray-200">
+      <img class=" w-4 h-4" src="./img/dots-menu.png" alt="">
+    </div>
+     <div class="p-2 hover:rounded-full hover:bg-gray-200">
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor"
+        class="w-6">
+        <path stroke-linecap="round" stroke-linejoin="round"
+          d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z" />
+      </svg>
+     </div>
+    
+  </div>
+  <div class="flex lg:hidden justify-between  text-gray-500 px-10 pt-8 pb-20 text-sm ">
+    <div class="flex gap-4 items-center">
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor"
+        class="w-6 h-6">
+        <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+      </svg>
+      <a href class="hover:text-blue-800 hover:underline">ALL</a>
+      <a href class="hover:text-blue-800 hover:underline">IMAGES</a>
+    </div>
+    <div class="flex gap-4 items-center">
+      <img class="w-4" src="./img/dots-menu.png" alt="">
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor"
+        class="w-6 h-6">
+        <path stroke-linecap="round" stroke-linejoin="round"
+          d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z" />
+      </svg>
+    </div>
+  </div>
+ 
+  <!-- main -->
+  <div class=" grow flex  flex-col items-center gap-6 ">
+    <img class="w-48 lg:w-64" src="./img/googlelogo_color_272x92dp.png" alt="">
+    <div
+      class=" w-[90%] lg:w-[40%] flex  items-center pt-1 pl-4 pr-2 rounded-full border hover:shadow-md hover:border-none">
```

Comment:
it's not recommended to use arbitrary value , you should add them in tailwind configuration file if needed